{% extends no_auth_sitebase.html %}
{% block content %}
<h2>{% raw text_locale['FAQ_HEADER'] %}</h2>
<div class="faq_wrapper">
  <div class="faq">
      <div class="separator"></div>
      <h3>{% raw text_locale['WATCH_VIDEOS'] %}</h3>
      <div class="videos clearfix">
      <div class="video"><iframe src="{{media_locale['steps_video']}}" width="330" height="200" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe> <p>{% raw text_locale['STEPS_TO_FOLLOW'] %}</p></div>
      <div class="video"><iframe src="http://player.vimeo.com/video/63926337" width="330" height="200" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe> <p>{% raw text_locale['LOG_FOOD']%}</p></div>
      <div class="video"><iframe src="http://player.vimeo.com/video/63931218" width="330" height="200" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe> <p>{% raw text_locale['ADD_PARTICIPANT'] %}</p></div>
      </div>
      <br />
      <div class="separator"></div>
      <br>
      <h3>{% raw text_locale['LOOK_BELOW'] %}</h3>
      <ul class="faq">
      <li><a href="#faq0">{% raw text_locale['PROJECT_101'] %}</a></li>
      <li><a href="#faq1">{% raw text_locale['PASSWORD_SAME_VERIFICATION'] %}</a></li>
      <li><a href="#faq2">{% raw text_locale['PASSWORD_DOESNT_WORK'] %}</a></li>
      <li><a href="#faq3">{% raw text_locale['LOGGED_IN_WHAT_NOW'] %}</a></li>
      <li><a href="#faq4">{% raw text_locale['TAKES_SIX_MONTHS'] %}</a></li>
      <li><a href="#faq5">Why are there 2 swabs inside the tube?</a></li>
      <li><a href="#faq6">{% raw text_locale['WHY_TWO_SWABS'] %}</a></li>
      <li><a href="#faq7">{% raw text_locale['WHAT_FORMS'] %}</a></li>
      <li><a href="#faq8">{% raw text_locale['RAW_DATA'] %}</a></li>
      <li><a href="#faq9">{% raw text_locale['WHERE_SEND_SAMPLE'] %}</a></li>
      <li><a href="#faq10">{% raw text_locale['DIFFERENT_WHATS_WRONG_WITH_ME'] %}</a></li>
      <li><a href="#faq11">{% raw text_locale['BETTER_OR_WORSE'] %}</a></li>
      <li><a href="#faq12">{% raw text_locale['WHEN_RESULTS_NON_FECAL'] %}</a></li>
      <li><a href="#faq13">{% raw text_locale['ONLY_FECAL_RESULTS'] %}</a></li>
      <li><a href="#faq14">{% raw text_locale['COMBINE_RESULTS'] %}</a></li>
      <li><a href="#faq15">{% raw text_locale['FECAL_NO_RESULTS'] %}</a></li>
      <li><a href="#faq16">{% raw text_locale['INFORMATION_IDENTIFY_ME'] %}</a></li>
      <li><a href="#faq17">{% raw text_locale['FIND_DETAILED_INFO'] %}</a></li>
      <li><a href="#faq18">{% raw text_locale['WHO_MICHAEL_POLLAN'] %}</a></li>
      <li><a href="#faq19">{% raw text_locale['HOW_CHANGE_GUT'] %}</a></li>
      <li><a href="#faq20">{% raw text_locale['ANOTHER_COPY_RESULTS'] %}</a></li>
      <li><a href="#faq21">{% raw text_locale['FOOD_DIARY'] %}</a></li>
      <li><a href="#faq22">{% raw text_locale['CONVERT_GRAMS_CALORIES'] %}</a></li>
      <li><a href="#faq23">{% raw text_locale['PARTICIPATE_WITH_DIAGNOSIS'] %}</a></li>
      <li><a href="#faq24">{% raw text_locale['MISSING_METADATA'] %}</a></li>
      <li><a href="#faq25">{% raw text_locale['HOW_PROCESS_SAMPLES'] %}</a></li>
      <li><a href="#faq26">{% raw text_locale['NOT_A_BUSINESS'] %}</a></li>
      </ul>
      <br>
      <div class="separator"></div>
      <br>

      <h3 class="faq_header" id="faq0">{% raw text_locale['PROJECT_101'] %}</h3>
      <div class="faq_content">
            <p>{% raw text_locale['INTRODUCTION_BEGINNING'] %}</p>
            <div class="indent_box">
                  {% raw text_locale['INTRODUCTION_TEXT'] %}
            </div>
      </div>
      <br>
      <div class="separator"></div>
      <br>
      <h3 class="faq_header" id="faq1">{% raw text_locale['PROJECT_101'] %}</h3>
      <div class="faq_content">
          <p>No. Your <strong>password</strong> is printed on the sheet that you received with your kit in the mail. That sheet looks like this:</p><img src="/static/img/creds_example.png"/><p>Your <strong>verification code</strong> is emailed to you. Look for the email: <br /><br />
<strong>FROM:</strong>  American Gut Project (info@americangut.org)<br />
<strong>SUBJECT:</strong>  American Gut Kit ID & Verification Code</p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq2">My password doesn't work!</h3>
      <div class="faq_content">
          <p>The passwords have some ambiguous characters in them, so we have this guide to help you decipher which characters are in your password.</p>
          <p class="ambig">
          abcdefghijklmnopqrstuvwxyz<br>
          ABCDEFGHIJKLMNOPQRSTUVWXYZ<br>
          1234567890<br>
          1 = the number 1<br>
          l = the letter l as in Lima<br>
          0 = the number 0<br>
          O = the letter O as in Oscar<br>
          g = the letter g as in golf<br>
          q = the letter q as in quebec
          </p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq3">I'm logged in, what do I do now?</h3>
      <div class="faq_content">
          <p>You need to follow the add participant workflow. Click on the "Add Source & Survey" tab located at the top of the page.</p>
          <img src="/static/img/add_participant.png">
          <p>During this workflow you (or whomever is being sampled) will:</p>
          <ol>
              <li>Add a participant</li>
              <li>Provide electronic consent</li>
              <li>Answer survey questions (including the diet questions covered by the food diary)</li>
              <li>Upon completion, become eligible to log samples</li>
          </ol>
          <p>When participants are eligible,  you will then see their name under the corresponding menu on the left, in this example we have just added the participant "Test":</p>
          <img src="/static/img/add_participant_menu.png"/>
          <p>You can log a sample by clicking the "Log Sample" link in the menu. If you do not see the "Log Sample" link, then all of your barcodes have been assigned.</p>
          <img src="/static/img/log_sample_options.png"/>
          <p>The generic add sample page looks like this:</p>
          <img src="/static/img/add_sample_highlight.png"/>
          <p>When adding a sample, please be sure to select the barcodes that matches the barcode on the sampling tube of the sample that you are logging</p>
          <img src="/static/img/add_sample_overview.png"/>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq4">Does it really take up to 6 months to get my results?</h3>
      <div class="faq_content">
          <p>Yes. It takes about 8 weeks for extractions, 8 weeks for the remainder of the processing, and 2 weeks to do the actual sequencing. This is before any analysis and if everything goes as planned, with no delays - equipment down, run failures, reagents or other consumables back ordered. Things do sometimes go wrong, so we say up to 6 months.</p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq5">Why are there 2 swabs inside the tube?</h3>
      <div class="faq_content">
      <p>Each sampling tube contains two swabs and looks like this:</p>
      <img src="/static/img/sample_barcode.jpg"/>
      <p>Each tube is used for <strong>one sample</strong>. The tube has two swabs in it because one is a backup in case the DNA does not amplify on the first swab.</p>
      <p>Here's a video of Rob Knight talking about swab handling:</p>
      <iframe src="http://player.vimeo.com/video/62393487" width="500" height="281" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe> <p><a href="http://vimeo.com/62393487">American Gut - Handling Your SWABS</a> from <a href="http://vimeo.com/user16100300">shelley schlender</a> on <a href="http://vimeo.com">Vimeo</a>.</p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq6">I have a 2+ sample kit, and would like to collect and send them in at different times&hellip;</h3>
      <div class="faq_content">
      <p>For best results, we recommend that you mail each sample within 24 hours of collection. </p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq7">What are the forms you talk about on the sampling instructions?</h3>
      <div class="faq_content">
      <p>The instruction on the sampling instructions that requires you to "place your forms and the sample tube in preaddressed envelope" is leftover from a previous version of the sampling instructions. There are no forms for you to include inside the envelope with your sample. If you are shipping internationally, please visit the <a href="/international_shipping/">International Shipping Instructions</a></p>
      </div>
      <div class="separator"></div>
      <h3 class="faq_header" id="faq8">How can I get the raw data?</h3>
      <div class="faq_content">
          <p>The raw data can be fetched from the <a href=http://www.ebi.ac.uk/>European Bioinformatics Institute</a>. EBI is part of <a href=http://www.insdc.org/>The International Nucleotide Sequence Database Collaboration</a> and is a public warehouse for sequence data. The deposited American Gut Project accessions so far are:</p>
          <ol>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003819&display=html">ERP003819</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003822&display=html">ERP003822</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003820&display=html">ERP003820</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP003821&display=html">ERP003821</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP005367&display=html">ERP005367</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP005366&display=html">ERP005366</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP005361&display=html">ERP005361</a></li>
              <li style="list-style-type:square"><a href="http://www.ebi.ac.uk/ena/data/view/ERP005362&display=html">ERP005362</a></li>
          </ol>
          <p>Processed sequence data and open-access descriptions of the bioinformatic processing can be found at our <a href="https://github.com/qiime/American-Gut">Github repository</a>.</p>
          <p>Sequencing of American Gut samples is an on-going project, as are the bioinformatic analyses. These resources will be updated as more information is added and as more open-access descriptions are finalized.</p>
      </div>
      <h3 class="faq_header" id="faq9">Where do I send my sample?</h3>
      <div class="faq_content">
          <p>This is the shipping address:</p>
              American Gut Project
              <br>Knight Lab, JSCBB
              <br>596 UCB
              <br>Boulder, CO 80309
          <p>If you are shipping internationally, please see the <a href="/international_shipping/">international shipping instructions</a>.

      <div class="separator"></div>
      </div>
      <h3 class="faq_header" id="faq10">I'm different than other people in my category. Does that mean something is wrong with me?</h3>
      <div class="faq_content">
          <p>No! Your gut microbiome is as unique as your fingerprint so you should expect to see some differences. Many factors can affect your gut microbiome, and any differences you see are likely to be the result of one of these factors. Maybe your diet is different than most people your age. Maybe you just traveled somewhere exotic. Different does not necessarily mean bad.</p>
      </div>
      <div class="separator"></div>


      <h3 class="faq_header" id="faq11">How can I tell if my gut microbiome is better or worse than other people in my category?</h3>
      <div class="faq_content">
          <p>Right now, you can’t. We’re still trying to understand what constitutes a normal or average gut microbiome, and we have a lot to learn about the functions of many of the microbes that inhabit the gut. Therefore, it’s tough to know what combinations of microbes are best for nutrition and health. That’s one reason collecting data from so many people is important—hopefully we can start to learn more about this.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq12">I sent in a non-fecal sample and have not received any results, when should I expect results?</h3>
      <div class="faq_content">
          <p>The vast majority of the samples we’ve received are fecal, which was why we prioritized those samples. Much of the analysis and results infrastructure we’ve put in place is applicable to other sample types, but we do still need to assess what specific representations of the data make the most sense to return to participants. We apologize for the delay. Our tentative goal for skin and oral samples is January 1st, 2014, and environmental samples sometime during the first quarter of 2014. </p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq13">I sent more than one kind of sample, but I only received data for my fecal sample. What happened to my other samples?</h3>
      <div class="faq_content">
          <p>We have only sent out results for fecal samples and are in the process of evaluating how best to present the other sample types. Please see <a href="#faq12">the previous question </a></p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq14">My whole family participated, can we combine the results somehow?</h3>
      <div class="faq_content">
          <p>We're still evaluating how best to present the data for samples that represent a family. We are mailing individual results now and will provide updated results through the web site later.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq15">I sent in a fecal sample but did not get any results, what happened to them?</h3>
      <div class="faq_content">
          <p>On any given sequencing run (not just the American Gut), a small percentage of the samples fail for unknown reasons -- our methods are good but not perfect. This is one of the reasons the sample kits have two Q-tips. It allows us to perform a second microbial DNA extraction and re-sequence if the first attempt failed. We will be doing this for all of the samples that failed. If there was a technical problem with the sample itself (e.g. not enough microbes on the swab) that inhibits us from producing data for you, we will be re-contacting you about collecting another sample.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq16">Can data describing my gut microbiome be used to identify me or a medical condition I have?</h3>
      <div class="faq_content">
          <p>No. First, all of your personal information has been de-identified in our database as mandated by institutional guidelines. Second, although each person has a unique gut microbiome, many of the unique qualities are at the species or strain level of bacteria. Our sequencing methods currently do not allow us to describe your gut microbiome in that much detail. Finally, for most medical conditions, there are no known, predictable patterns in gut microbial community composition. Research simply hasn’t gotten that far yet.</p>
          <p>We should also mention that since we are only interested in your microbes, we do not sequence human genomic DNA in our typical analyses. Where it is possible for human DNA to be sequenced (e.g., the Beyond Bacteria kits), we remove the human DNA using the same bioinformatics approaches undertaken in the NIH-funded Human Microbiome Project and approved by NIH bioethicists. Additionally, there is so little human DNA in fecal, skin and mucus samples that the chances of us being able to sequence your entire human genome are almost none, even if we tried. </p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq17">Where can I find more detailed information about my sample?</h3>
      <div class="faq_content">
          <p>You can find the raw data from European Bioinformatics Institute (please see <a href="#faq8">here</a>) or download the copy of your result from our website (please see <a href="#faq20">here</a>).</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq18">Who is Michael Pollan?</h3>
      <div class="faq_content">
          <p>Michael Pollan is a New York Times Best Seller for his books on diet and nutrition. Further information about Michael can be found <a href="http://michaelpollan.com/">here</a>. </p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq19">How can I change my gut microbiome?</h3>
      <div class="faq_content">
          <p>Although we still don’t have a predictable way to change the gut microbiome in terms of increasing or decreasing the abundances of specific bacteria, we do know that a variety of factors influence gut microbial community composition. Diet is a major factor affecting the gut microbiome so by changing your diet, you may be able to affect your gut microbiome. We still don’t fully understand probiotics but know that they can influence your gut microbiome while you are actively taking them. Factors such as stress can also influence the gut microbiome. However, it is important to remember that there are factors we can’t change, such as age or genetics, that can affect the gut microbiome.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq20">Can I get another copy of my results?</h3>
      <div class="faq_content">
          <p>You can download a copy from our website. Log in with your account name and password, go to the left side bar, move your mouse to Human Samples -> PARTICIPANT NAME -> SAMPLE NUMBER, and then click on SAMPLE NUMBER.pdf to download it.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq21">Why am I asked for a seven-day food diary? Do I really need to fill the survey?</h3>
      <div class="faq_content">
          <p>In fact, responses to all questions on the survey, including the dietary questions, are strictly voluntary. Technically you are not required to provide all those information, but it is <em>highly recommended</em>. Having the information will help us make possible connections from your life styles to your microbe living on/within you and hopefully figure out how that can impact your health. Particularly, we know diet is a major factor affecting the gut microbiome since our gut microbes eat the food we don’t digest. Having diet information will allow us to look for patterns in gut microbiome composition across populations of people with different diets. </p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq22">How do I convert grams of macronutrients into percentage of calories?</h3>
      <div class="faq_content">
          <p>Once you determine how many grams of protein, carbohydrates and lipids you have consumed each day, multiply the grams of protein and the grams of carbohydrates by 4 each (since there are 4 calories in every gram of protein and carbohydrate) and multiply the grams of lipids by 9 (since there are 9 calories in every gram of lipids). Add all of these numbers together to determine the total number of calories you consumed. Divide protein calories by the total calories to determine the percentage of calories consumed as protein. Repeat for carbohydrates and lipids.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq23">Can I participate in the project if I am diagnosed with ….?</h3>
      <div class="faq_content">
          <p>Of course! The only exclusion criteria are: you must be more than 6 weeks old and cannot be a convicted felon. Please keep in mind that, for legal and ethical reasons, the American Gut Project does not provide medically actionable results or advice.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq24">What are missing metadata?</h3>
      <div class="faq_content">
          <p>Metadata are information describing your age, gender, diet, etc. Missing metadata mean that this person did not provide us with this information.</p>
      </div>
      <div class="separator"></div>

      <h3 class="faq_header" id="faq25">How are the samples and data processed?</h3>
      <div class="faq_content">
          <p>The majority of the samples in the American Gut
          Project are run through a processing pipeline
          designed to amplify a small region of a gene that
          is believed to be common to all Bacteria and
          Archaea. This gene, the 16S ribosomal RNA gene is
          like a barcode you find on your groceries, and
          serves as a marker for different organisms. There
          are quite a few different ways to assess the types
          of Bacteria and Archaea in a sample, including a
          variety of techniques even to look at this single
          gene. Every method has its biases, and comparing
          data between different methods is <a
              href="http://www.ncbi.nlm.nih.gov/pubmed/23861384">non-trivial</a>
          and can sometimes be nearly impossible. One of the
          primary goals of the American Gut is to provide
          data that can be used and reused by researchers
          worldwide, we have opted to use the standard
          protocols adopted by the <a
              href="http://earthmicrobiome.org">Earth
              Microbiome Project</a>, (<a
              href="http://www.ncbi.nlm.nih.gov/pubmed/22402401">Caporaso
              et al 2012</a>, and more detailed description
          of the <a
              href="http://www.earthmicrobiome.org/emp-standard-protocols/16s/">protocol</a>).
          This ensures that the data generated by the
          American Gut can be combined with the other 80,000
          samples so far indexed by the EMP (as scientists,
          we get giddy about things like this).</p>
          
          <p>DNA sequencing is a complex challenge that
          involves an army of robots, ultra pure water that
          costs $75 per 10ml, and an amazing <a
              href="http://www.illumina.com/systems/miseq.ilmn">digital
              camera</a> that actually determines individual
          sequences one nucleotide at a time. The number of
          stunningly brilliant minds whose footprints exist
          in these methods is astounding. However, the
          challenges don’t end once you get the DNA sequence
          - some might say they are just beginning. It turns
          out that figuring out what actually is in your
          sample, that is, what organisms these sequences
          correspond to, requires cutting edge computational
          approaches, supercomputers and caffeine for the
          people operating them. The questions being asked of
          the data are themselves complex, and volume of data
          being processed is simply phenomenal. To give you
          some idea, for each sample sequenced we obtain
          around 6 million nucleotides which we represent as
          letters (A, T, G or C, see <a
              href="http://en.wikipedia.org/wiki/Nucleotide">here</a>
          for more info), whereas Shakespeare's Hamlet only
          contains around 150,000 letters (ignoring
          spaces).</p>
          
          <p>The primary software package we use for
          processing 16S sequence data is called Quantitative
          Insights into Microbial Ecology (<a
              href="http://www.qiime.org">QIIME</a>; <a
              href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">Caporaso
              et al. 2010</a>). Using this package, we are
          able to start with raw sequence data and process it
          to so that we end up be able to  explore the
          relationships within and between samples using a
          variety of statistical methods and metrics. To help
          in the process, we leverage a standard and
          comprehensive (to date) reference database called
          Greengenes (<a
              href="http://www.ncbi.nlm.nih.gov/pubmed/22134646">McDonald
              et al. 2011</a>; <a
              href="http://www.ncbi.nlm.nih.gov/pubmed/16820507">DeSantis
              et al. 2006</a>) that includes information on a
          few hundred thousand Bacteria and Archaea (it is
          likely that there are millions or more species of
          bacteria). Due to the molecular limitations of our
          approach, and the lack of a complete reference
          database (because the total diversity of microbes
          on Earth is still unknown), our ability to
          determine whether a specific organism is present
          has a margin of error on the order of millions of
          years, which limits our ability to assess specific
          strains or even species using this inexpensive
          technique (more expensive techniques, such as some
          of the higher-level perks, can provide this
          information). But all is not lost! By using the
          evolutionary history of the organisms as inferred
          by the small pieces of DNA that we have, we can
          begin to ask broad questions about the diversity
          within (see <a
              href="http://www.ncbi.nlm.nih.gov/pubmed/7972354">Faith
              1994</a>) and between samples (see <a
              href="http://www.ncbi.nlm.nih.gov/pubmed/16332807">Lozupone
              and Knight 2005</a>), and whether the patterns
          observed relate to study variables (e.g., BMI,
          exercise frequency, etc).</p>
          
          <p>The specifics on how the American Gut sequence
          data are processed can be found <a
              href="http://nbviewer.ipython.org/github/biocore/American-Gut/blob/master/ipynb/module2_v1.0.ipynb">here</a>,
          and are written up in an executable <a
              href="http://ipython.org/notebook">IPython
              Notebook</a>, which provides all the relevant
          processing steps in an open-source format. Be
          warned, processing the full American Gut dataset
          takes over 5,000 CPU hours right now (i.e. if you
          do it on your laptop it might take 7 months, even
          if you don’t run out of memory: this might put the
          time it takes to get your results in perspective).
          This is the processing pipeline that we use on your
          data. As this project is a work in progress, we are
          versioning the processing pipeline as there will
          continue to be improvements to the process as the
          project moves forward.</p>

          <p>Additional information about the tools used in the American Gut Project and our contributions to the microbiome community can be found in the following publications:</p>

          <ul>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21552244">Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24280061">EMPeror: a tool for visualizing high-throughput microbial community data.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332807">UniFrac: a new phylogenetic method for comparing microbial communities.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16893466">UniFrac--an online tool for comparing microbial community diversity in a phylogenetic context.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220268">Quantitative and qualitative beta diversity measures lead to different insights into factors that structure microbial communities.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19710709">Fast UniFrac: facilitating high-throughput phylogenetic analyses of microbial communities including analysis of pyrosequencing and PhyloChip data.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20827291">UniFrac: an effective distance metric for microbial community comparison.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21885731">Linking long-term dietary patterns with gut microbial enterotypes.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23326225">A guide to enterotypes across the human body: meta-analysis of microbial community structures in human microbiome datasets.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699609">Structure, function and diversity of the healthy human microbiome.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699610">A framework for human microbiome research.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587224">The Biological Observation Matrix (BIOM) format or: how I learned to stop worrying and love the ome-ome.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22134646">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304728">The Earth Microbiome Project: Meeting report of the "1 EMP meeting on sample selection and acquisition" at Argonne National Laboratory October 6 2010.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304727">Meeting report: the terabase metagenomics workshop and the vision of an Earth microbiome project.</a></li>
          </ul>

          <p>More detail on our work on the effects of storage conditions can be found in these publications:</p>

          <ul>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20412303">Effect of storage conditions on the assessment of bacterial community structure in soil and human-associated samples.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20673359">Sampling and pyrosequencing methods for characterizing bacterial communities in the human gut using 16S sequence tags.</a></li>
          </ul>

          <p>And more detail on our work on sequencing and data analysis protocols can be found in these publications:</p>

          <ul>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17881377">Short pyrosequencing reads suffice for accurate microbial community analysis.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18723574">Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18264105">Error-correcting barcoded primers for pyrosequencing hundreds of samples in multiplex.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22237546">Selection of primers for optimal taxonomic classification of environmental 16S rRNA gene sequences.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22170427">Comparison of Illumina paired-end and single-direction sequencing for microbial 16S rRNA gene amplicon surveys.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21716311">Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21349862">PrimerProspector: de novo design and taxonomic analysis of barcoded polymerase chain reaction primers.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">QIIME allows analysis of high-throughput community sequencing data.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22161565">Using QIIME to analyze 16S rRNA gene sequences from microbial communities.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23861384">Meta-analyses of studies of the human microbiota.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24060131">Advancing our understanding of the human microbiome using QIIME.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20534432">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22402401">Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq platforms.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23202435">Quality-filtering vastly improves diversity estimates from Illumina amplicon sequencing.</a></li>
          <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699611">Human gut microbiome viewed across age and geography.</a></li>
          </ul>
      </div>
      
      <h3 class="faq_header" id="faq26">We are not a business</h3>
      <div class="faq_content">

          <p>We have had many enquiries about our "service"
          or "business". American Gut is a donation-supported
          academic project that is a collaboration between
          the <a href="http://www.earthmicrobiome.org">Earth
              Microbiome Project</a> and the <a
              href="http://humanfoodproject.com/">Human Food
              Project</a>, primarily run out of the <a
              href="https://knightlab.colorado.edu/">Knight
              Lab</a> at the University of Colorado at
          Boulder, and is not a business or service.  In
          particular, American Gut is not a diagnostic test
          (although the information gained through the
          project may in future contribute to the development
          of diagnostic tests). All data except for
          information that needs to be kept confidential for
          privacy reasons is openly and freely released into
          public databases, and the project is not intended
          to make a profit (any surplus funds would be
          recycled back into furthering human microbiome
          research).</p>
      </div>
      <div class="separator"></div>
  </div>
</div>
{% end %}
